cmd/commands/destroy_cluster.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"fmt"
+	"github.com/rook/kubectl-rook-ceph/pkg/crds"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+	"github.com/spf13/cobra"
+	"os"
+	"strings"
+	"time"
+)
+
+const (
+	destroyClusterQuestion = "Are you sure you want to destroy the cluster in namespace %q? If absolutely certain, enter: " + destroyClusterAnswer
+	destroyClusterAnswer   = "yes-really-destroy-cluster"
+	toolBoxDeployment      = "rook-ceph-tools"
+	destroyProcessTimeOut  = 2 * time.Minute
+)
+
+// DestroyClusterCmd represents the command for destroy cluster
+var DestroyClusterCmd = &cobra.Command{
+	Use:   "destroy-cluster",
+	Short: "delete all Rook CRs",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := cmd.Context()
+		question := fmt.Sprintf(destroyClusterQuestion, CephClusterNamespace)
+		confirmation, err := confirmPrompt(question, destroyClusterAnswer)
+		if err != nil {
+			logging.Fatal(err)
+		}
+
+		logging.Info(confirmation)
+		clientsets := GetClientsets(ctx)
+		crds.DeleteCustomResources(ctx, clientsets, CephClusterNamespace)
+		k8sutil.DeleteDeployment(ctx, clientsets.Kube, CephClusterNamespace, toolBoxDeployment)
+		time.Sleep(destroyProcessTimeOut)
+		logging.Info("done")
```

Comment:
```suggestion
		k8sutil.DeleteDeployment(ctx, clientsets.Kube, CephClusterNamespace, toolBoxDeployment)
		time.Sleep(destroyProcessTimeOut)
		logging.Info("done")
```

IMO above lines can be moved to crds.go file where we are removing the resources

Reply:
yea totally agree, I already did this change